### PR TITLE
Cohesity partner addition

### PR DESCRIPTION
**Pull request by denneyliptak**
Greetings Michael,

We just got the predictive URL for the blog and I've added it to the page.
May I request your review of the changes to add the Cohesity 'Partner' page?

The blog goes live on Friday and Parissa has a reference to the SCOD page so we'll want to get it final by then.

Thank you much,
--Denney

### CODE REVIEW

File: docs/partners/cohesity/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Cohesity
+Hewlett Packard Enterprise and Cohesity offer an integrated approach to solve customer problems commonly found with containerized workloads. HPE Alletra—leveraging the HPE CSI Driver for Kubernetes—together with Cohesity's comprehensive data protection capabilities, empower organizations to overcome challenges associated with containerized environments.
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:2]: For readability, add a blank line after headers.

File: docs/partners/cohesity/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Review Cohesity's "[Register and Manage Kubernetes Cluster](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/register.htm?tocpath=Kubernetes%7C_____2#RegisterKubernetesClusterasaSource)" documentation to integrate Cohesity into your Kubernetes cluster. Below is an example screenshot of the <b>Register Kubernetes Source</b> dialog:<br />
+![](img/register_k8s.png) <br /> 
+
+After the integration wizard is submitted, see the [Post-Registration task documentation](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/register.htm?tocpath=Kubernetes%7C_____2#PostRegistrationTask) to verify Velero and datamover pod availability.
+
+!!! note
+    The latest versions of Kubernetes, although present in the [Cohesity support matrix](https://docs.cohesity.com/7_0/Web/UserGuide/Content/ReleaseNotes/SupportedVersions.htm#Kubernet), may still require an override from Cohesity support.  
+
+### Configure Namespace-level Application Backup
+A Namespace containing a WordPress application will be protected in this example. It contains a variety of Kubernetes resources and objects including:
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:46]: For readability, add a blank line after headers.

File: docs/partners/cohesity/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+* Workloads: `Deployment`, `ReplicaSet` and `StatefulSet`
+
+Review the [Protect Kubernetes Namespaces](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/protect.htm?tocpath=Kubernetes%7CBackup%7C_____1) documentation from Cohesity. Create a new protection policy or use an available default policy. Additionally, see the [Manage the Kubernetes Backup Configuration](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/manage-backup.htm) documentation to add/remove namespaces to a protection group, adjust Auto Protect settings, modify the Protection Policy, and trigger an on-demand run.
+
+See the screenshot below for an example backup <b>Run details</b> view.<br/>
+![](img/Cohesity_Protection-RunDetails-view.png)
+
+
+### Demo: Clone a Test/Development Environment by Restoring a Backup
+Review the Cohesity documentation for [Recover Kubernetes Cluster](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/ContainerRecover.htm?tocpath=Kubernetes%7C_____5#RecoverKubernetesCluster). Cohesity notes, at time of writing, that granular-level recovery of namespaces is not supported. Consider the following when defining a recovery operation:
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:59]: For readability, add a blank line after headers.

File: docs/partners/cohesity/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+* If an alternate Kubernetes cluster is chosen (<b>New Location</b> in the UI), be sure that the cluster has access to the same Kubernetes `StorageClass` as the backup’s source cluster.
+
+!!! note
+    Protection groups and individual namespaces appear in the same list. Namespaces are denoted with the Kubernetes ship wheel icon.
+
+For this example, a WordPress namespace backup will be restored to the source Kubernetes cluster but under a new namespace with a "debug-" prefix (see below). This application can run alongside and separately from the parent application.
+![](img/Cohesity-Recovery-Namespace-locationandrename.png)
+
+After the recovery process is complete we can review and compare the associated objects between the two namespaces. In particular, names are similar but discrete volumes, IPs and services exist for each namespace.
+```text
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:73]: For readability, add a blank line between paragraphs and markdown elements.

File: docs/partners/cohesity/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+The HPE CSI Driver has been validated on Cohesity DataProtect v7.0u1. 
+Check that the [HPE CSI Driver](https://scod.hpedev.io/csi_driver/index.html#compatibility_and_support) and [Cohesity software](https://docs.cohesity.com/7_0/Web/UserGuide/Content/ReleaseNotes/SupportedVersions.htm#Kubernet) versions are compatible with the Kubernetes version being used.
+
+This environment assumes the HPE CSI Driver for Kuberentes is [deployed](../../csi_driver/deployment.md) in the Kubernetes cluster, an Alletra storage [backend](../../csi_driver/deployment.md#add_an_hpe_storage_backend) has been configured, and a default [`StorageClass`](../../csi_driver/using.md#base_storageclass_parameters) has been defined.
+
+Review Cohesity's "[Plan and Prepare](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/plan-prepare.htm)" documentation to accomplish the following:
+
+* Firewall considerations.
+* Kubernetes `ServiceAccount` with `cluster-admin` persissions.
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:29]: Speling

File: docs/partners/cohesity/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+* Select a protection group or individual namespace. If a protection group is chosen, multiple namespaces could be affected on recovery.
+* If any previously backed up objects exist in the destination, a restore operation <i>will not</i> overwrite them. 
+* For applications deployed by Helm chart, recovery operations applied to new clusters or namespaces will not be managed with Helm.
+* If an alternate Kubernetes cluster is chosen (<b>New Location</b> in the UI), be sure that the cluster has access to the same Kubernetes `StorageClass` as the backup’s source cluster.
+
+!!! note
+    Protection groups and individual namespaces appear in the same list. Namespaces are denoted with the Kubernetes ship wheel icon.
+
+For this example, a WordPress namespace backup will be restored to the source Kubernetes cluster but under a new namespace with a "debug-" prefix (see below). This application can run alongside and separately from the parent application.
+![](img/Cohesity-Recovery-Namespace-locationandrename.png)
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:70]: Add a line break here, it looks a bit tight in the final rendering.

File: docs/partners/cohesity/index.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+A Namespace containing a WordPress application will be protected in this example. It contains a variety of Kubernetes resources and objects including:
+
+* Configuration and Storage: `PersistentVolumeClaim`, `ConfigMap`, and `Secret` 
+* `Service` and `ServiceAccount`
+* Workloads: `Deployment`, `ReplicaSet` and `StatefulSet`
+
+Review the [Protect Kubernetes Namespaces](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/protect.htm?tocpath=Kubernetes%7CBackup%7C_____1) documentation from Cohesity. Create a new protection policy or use an available default policy. Additionally, see the [Manage the Kubernetes Backup Configuration](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/manage-backup.htm) documentation to add/remove namespaces to a protection group, adjust Auto Protect settings, modify the Protection Policy, and trigger an on-demand run.
+
+See the screenshot below for an example backup <b>Run details</b> view.<br/>
+![](img/Cohesity_Protection-RunDetails-view.png)
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:55]: Add a line-break between these.

File: docs/partners/cohesity/index.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+* Kubernetes `ServiceAccount` with `cluster-admin` persissions.
+* Extract Bearer token ID from above `ServiceAccount`
+* Obtain Cohesity Datamover [(download)](https://downloads.cohesity.com/oauth2/login) and push to a local repository or public registry.
+!!! note
+    Cohesity only supports the backup of user-created application namespaces and does not support the backup of infrastructure namespaces such as `kube-system`, etc.
+
+### Integrate Cohesity into Kubernetes
+
+Review Cohesity's "[Register and Manage Kubernetes Cluster](https://docs.cohesity.com/7_0/Web/UserGuide/Content/Container/register.htm?tocpath=Kubernetes%7C_____2#RegisterKubernetesClusterasaSource)" documentation to integrate Cohesity into your Kubernetes cluster. Below is an example screenshot of the <b>Register Kubernetes Source</b> dialog:<br />
+![](img/register_k8s.png) <br />
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:38]: Add a line-break between these.

File: docs/partners/cohesity/index.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
+# Cohesity
+Hewlett Packard Enterprise and Cohesity offer an integrated approach to solve customer problems commonly found with containerized workloads. HPE Alletra—leveraging the HPE CSI Driver for Kubernetes—together with Cohesity's comprehensive data protection capabilities, empower organizations to overcome challenges associated with containerized environments. 
+
+This guide will demonstrate the steps to integrate Cohesity into a Kubernetes cluster and how to configure a protection policy to back up an application namespace. It proceeds to show that a backup can be restored to a new namespace, useful for providing a test/development environment without affecting the original application namespace.
+
+External Resources:
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:6]: External HPE Resources?

File: docs/partners/cohesity/index.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+
+!!! note
+    Protection groups and individual namespaces appear in the same list. Namespaces are denoted with the Kubernetes ship wheel icon.
+
+For this example, a WordPress namespace backup will be restored to the source Kubernetes cluster but under a new namespace with a "debug-" prefix (see below). This application can run alongside and separately from the parent application.
+![](img/Cohesity-Recovery-Namespace-locationandrename.png)
+
+After the recovery process is complete we can review and compare the associated objects between the two namespaces. In particular, names are similar but discrete volumes, IPs and services exist for each namespace.
+```text
+$ diff <(kubectl get all,pvc -n wordpress-orig) <(kubectl get all,pvc -n debug-wordpress-orig)
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:74]: This `diff` output looks a bit messy. What if we put on block after another instead?

I suspect `diff --side-by-side` will be too wide?

File: docs/partners/cohesity/index.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+* For applications deployed by Helm chart, recovery operations applied to new clusters or namespaces will not be managed with Helm.
+* If an alternate Kubernetes cluster is chosen (<b>New Location</b> in the UI), be sure that the cluster has access to the same Kubernetes `StorageClass` as the backup’s source cluster.
+
+!!! note
+    Protection groups and individual namespaces appear in the same list. Namespaces are denoted with the Kubernetes ship wheel icon.
+
+For this example, a WordPress namespace backup will be restored to the source Kubernetes cluster but under a new namespace with a "debug-" prefix (see below). This application can run alongside and separately from the parent application.
+![](img/Cohesity-Recovery-Namespace-locationandrename.png)
+
+After the recovery process is complete we can review and compare the associated objects between the two namespaces. In particular, names are similar but discrete volumes, IPs and services exist for each namespace.
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:72]: In general, on SCOD, when we call out Kubernetes resources, we monospace those. So this line would read:

After the recovery process is complete we can review and compare the associated objects between the two `Namespaces`. In particular, names are similar but discrete `PersistentVolumes`, IPs and `Services` exist for each `Namespace`.

**Review by datamattsson — CHANGES_REQUESTED** [review]
A few nits here and there.

File: docs/partners/cohesity/index.md (hunk 10 above)
Comment (denneyliptak) [reply docs/partners/cohesity/index.md:81]: Was able to modify the syntax a bit so that mkdocs will help color. This helps improve, yea?

**Review by datamattsson — CHANGES_REQUESTED** [review]
Can you update `Namespaces` where appropriate?

File: docs/partners/cohesity/index.md [hunk 12]
```diff
@@ -61,17 +61,17 @@ See the screenshot below for an example backup <b>Run details</b> view.<br/>
...
+* For applications deployed by Helm chart, recovery operations applied to new clusters or `Namespaces` will not be managed with Helm.
 * If an alternate Kubernetes cluster is chosen (<b>New Location</b> in the UI), be sure that the cluster has access to the same Kubernetes `StorageClass` as the backup’s source cluster.
 
 !!! note
-    Protection groups and individual namespaces appear in the same list. Namespaces are denoted with the Kubernetes ship wheel icon.
+    Protection groups and individual `Namespace` resources appear in the same list. Available `Namespaces` are denoted with the Kubernetes ship wheel icon.
 
-For this example, a WordPress namespace backup will be restored to the source Kubernetes cluster but under a new namespace with a "debug-" prefix (see below). This application can run alongside and separately from the parent application.
+For this example, a WordPress `Namespace` backup will be restored to the source Kubernetes cluster but under a new `Namespace` with a `debug-` prefix (see below). This application can run alongside and separately from the parent application.
```
Comment (datamattsson) [inline docs/partners/cohesity/index.md:74]: leave "debug-" as-is, that's a name, not a resource kind.

**Review by datamattsson — CHANGES_REQUESTED** [review]
Last nit.

**Review by datamattsson — APPROVED** [review]

**Pull request merged**